change/@fluentui-react-tabs-2020-09-22-11-36-27-pivot-hc-styling.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Update Pivot.scss in react-tabs package",
+  "packageName": "@fluentui/react-tabs",
```

Comment:
you are missing the change for `Pivot.styles.ts` in `react-tabs` package. that one needs to be in sync with OUFR